src/features/CareViewPatientsSummary/components/SlotDetailsCell.jsx

```diff
@@ -84,7 +86,11 @@ export const SlotDetailsCell = ({
       if (slotItem.isNonMedication) {
         const taskItem = slotItem;
         return (
-          <div className="slot-details" key={`${taskItem.uuid}`}>
+          <div
+            className="slot-details"
+            key={`${taskItem.uuid}`}
+            style={{ borderColor: "#0f62fe", background: "#e1f3fe" }}
```

Comment:
Why are we adding inline styles though we have added class ? Can we keep it consistent by defining a class ?

Reply:
Sure Mohan, makes sense. Will do.